src/main/scala/cognite/spark/v1/CdfSparkAuth.scala

```diff
@@ -2,30 +2,29 @@ package cognite.spark.v1
 
 import cats.effect._
 import com.cognite.sdk.scala.common.{Auth, AuthProvider, OAuth2}
-import com.softwaremill.sttp.SttpBackend
-import com.softwaremill.sttp.asynchttpclient.cats.AsyncHttpClientCatsBackend
+import sttp.client3.SttpBackend
 
 sealed trait CdfSparkAuth extends Serializable {
   def provider(
       implicit cs: ContextShift[IO],
       clock: Clock[IO],
-      sttpBackend: SttpBackend[IO, Nothing]): IO[AuthProvider[IO]]
+      sttpBackend: SttpBackend[IO, Any]): IO[AuthProvider[IO]]
```

Comment:
is this required by new sttp version?

Reply:
Yes, the new Sttp version has a different model of the second type parameter. I don't really understand what does it do, but Scala SDK also switched everything to Any from Nothing.